src/components/pages/UserManagement/UserManagement.js

```diff
@@ -54,25 +54,6 @@ const UserManagement = ({ allMentors, allMentees }) => {
       title: 'Role',
```

Comment:
Lines 35 and 51 sorter... are these required? what do they do and are they necessary. Are we sorting name or email because im not sure that we can sort both unless this is a feature that goes alongside of onFilter.. Is this something we can remove?

Reply:
I stuck to the context of the ticket and did not change these sections. I only updated the role section filter.